src/animatediff/utils/model.py

```diff
@@ -182,7 +182,8 @@ def get_motion_modules(
                 local_dir_use_symlinks=False,
                 resume_download=True,
             )
-            logger.debug(f"Downloaded {path_from_cwd(result)}")
+            path = Path(result)
+            logger.debug(f"Downloaded {path_from_cwd(path)}")
```

Comment:
I'm actually going to fix this one up in path_from_cwd() (by just wrapping the input path in `Path()` one more time, since that's a no-op if it's already a `Path`) - so I'll go ahead and drop this bit